sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py

```diff
@@ -724,7 +723,7 @@ def connect(self):
             self.ws = create_connection(
                 url="wss://{}".format(self._custom_endpoint or self._host),
                 subprotocols=[AMQP_WS_SUBPROTOCOL],
-                timeout=self._connect_timeout,
+                timeout=self.socket_timeout_interval,
```

Comment:
Websocket transport has no other read/write timeout configuration apart from "connect"?

Reply:
no, it doesn't use this anywhere else.